spec/controllers/api_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe(ApiController, type: :controller) do
+  describe 'GET #index' do
+    it 'responds successfully with an HTTP 200 status code' do
+      get :index
+      expect(response).to be_success
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the personal template' do
+      get :index
+      expect(response).to render_template('index')
+    end
+  end
+
+  describe 'POST #index' do
+   it 'responds successfully with an HTTP 200 status code' do
+      post :index
```

Comment:
you're usually posting with some parameters though, right? Might be good to test that as well